docs/install/index.md

```diff
@@ -63,314 +63,196 @@ Indicate your preferred configuration. Then, follow the customized commands to i
 <div class="cpu gpu">
 <div class="btn-group opt-group" role="group">
   <button type="button" class="btn btn-default environs opt active">Pip</button>
-  <button type="button" class="btn btn-default environs opt">Virtualenv</button>
   <button type="button" class="btn btn-default environs opt">Docker</button>
   <button type="button" class="btn btn-default environs opt">Build from Source</button>
 </div>
 </div>
 </div>
 </div>
-
+<hr>
 <!-- END - Main Menu -->
 
 <!-- START - Linux Python CPU Installation Instructions -->
 
 <div class="linux">
 <div class="python">
 <div class="cpu">
-
-The following installation instructions have been tested on Ubuntu 14.04 and 16.04.
-
-
 <div class="pip">
-<br/>
-
-**Step 1**  Install prerequisites - wget and latest pip.
-
-Installing *MXNet* with pip requires a latest version of `pip`. Install the latest version of `pip` by issuing the following command in the terminal.
-
-```bash
-$ sudo apt-get update
-$ sudo apt-get install -y wget python gcc
-$ wget https://bootstrap.pypa.io/get-pip.py && sudo python get-pip.py
-```
-
 <div class="v1-2-1">
 
-**Step 2** Install MXNet with OpenBLAS acceleration.
-
-```bash
-$ pip install mxnet
-```
-
-**Step 3**  Install [Graphviz](http://www.graphviz.org/). (Optional, needed for graph visualization using `mxnet.viz` package).
-```bash
-sudo apt-get install graphviz
-pip install graphviz
 ```
-
-**Step 4**  Validate the installation by running simple MXNet code described [here](#validate-mxnet-installation).
-
-**Experimental Choice** If You would like to install mxnet with Intel MKL, try the experimental pip package with MKL:
-```bash
-$ pip install mxnet-mkl
+$ pip install mxnet
```

Comment:
Not mxnet-mkl anymore? There are a lot of other options as well: https://pypi.org/search/?q=mxnet Maybe add some recommendation or at least list some options here?

Reply:
![2018-08-29_11-35-04](https://user-images.githubusercontent.com/5974205/44807904-998ab200-ab7f-11e8-8b90-66426b2eeec6.png)

I added the link to PyPI as a "footer" for every pip install instruction. That way users can look for them. I could also put in a footer note that adding *-mkl to a package would get them the experimental mkl support.... that would be good?

Reply:
Awesome